community/kernel/src/test/java/org/neo4j/kernel/impl/index/schema/fusion/SpatialFusionIndexPopulatorTest.java

```diff
@@ -64,7 +64,9 @@ public void setup() throws Exception
         SpatialCRSSchemaIndex.Supplier indexSupplier = mock( SpatialCRSSchemaIndex.Supplier.class );
         IndexDescriptor descriptor = IndexDescriptorFactory.forLabel( 42, 1337 );
 
-        for ( CoordinateReferenceSystem crs : asList( CoordinateReferenceSystem.WGS84, CoordinateReferenceSystem.Cartesian ) )
+        for ( CoordinateReferenceSystem crs : asList( CoordinateReferenceSystem.WGS84,
+                                                      CoordinateReferenceSystem.Cartesian,
+                                                      CoordinateReferenceSystem.Cartesian_3D ) )
```

Comment:
What is the reason WGS84-3D is excluded?

Reply:
No clue. It used to have a wrongly labelled `Cartesian_3D`, this addition is merely fixing a previously broken setup. 

This test will be deleted soon anyway by the spatial rewrite.